src/themes/common.tsx

```diff
@@ -13,9 +13,18 @@ export const dimensions = {
   },
   form: {
     button: {
-      small: '30px',
-      normal: '40px',
-      large: '60px'
+      height: {
+        extraSmall: '20px',
+        small: '30px',
+        normal: '40px',
+        large: '60px'
+      },
+      padding: {
+        extraSmall: '3px 10px',
```

Comment:
We shouldn't need `3px` here as the `line-height` is more reliable to center (and that seems to inherit the right value at the moment). Can we have this as `0` as per the other padding sizes?